docs/user-guide/00-introduction.md

```diff
@@ -1,3 +1,5 @@
 # Introduction
 
-This user guide refers primarily to the on [premises version of Image Builder](../on-premises/Overview). While many of the concepts are transferable to the service, at this point please refer to Red Hat's official documentation.
\ No newline at end of file
+With the Image Builder tool, you can create customized system images. Image builder automatically handles the setup details for each output type and is therefore easier to use and faster to work with than manual methods of image creation. You can access the Image Builder functionalities by using the command-line interface, or the Web UI interface. With Image Builder, you can create system images that are prepared for deployment, including system images prepared for clouds, and also images otimized for deployment on edge servers.
```

Comment:
```suggestion
With the Image Builder tool, you can create customized system images. Image builder automatically handles the setup details for each output type and is therefore easier to use and faster to work with than manual methods of image creation. You can access the Image Builder functionalities by using the command-line interface, or the Web UI interface. With Image Builder, you can create system images that are prepared for deployment, including system images prepared for clouds, and also images optimized for deployment on edge servers.
```

Reply:
Yay - first PR in this repo - welcome! :tada: 
And apart from the tiny typo, this looks good! :)

I'm wondering if more of the terms should become references to other pages (like "command-line interface", "clouds" or "edge servers"). WDYT?